WebisD/http-api-without-lib
# Feat creat Response class and Status Code class

**What was done**
* Create a Response class that will answer the requests received by the server
* Created a StatusCode enum class that for easy access of the most common status codes

**How to Test**
* Use *PUT*, *POST* or *DELETE* in postman and check the terminal. Like in this image
![image](https://user-images.githubusercontent.com/49571908/115423685-6c79ea80-a1d4-11eb-964f-3797a66b0399.png)
